ext/rubymain.cpp

```diff
@@ -1368,18 +1411,5 @@ extern "C" void Init_rubyeventmachine()
 
 	rb_define_method (EmConnection, "get_outbound_data_size", (VALUE(*)(...))conn_get_outbound_data_size, 0);
 	rb_define_method (EmConnection, "associate_callback_target", (VALUE(*)(...))conn_associate_callback_target, 1);
-
```

Comment:
It made no sense to define these Ruby constants from C! I moved them out to Ruby and made a hash that maps them to their symbols.